bounce/bin/main.ml

```diff
@@ -0,0 +1,120 @@
+open Claudius
+let () = Random.self_init ()
```

Comment:
Can you put this into the final `let () =` block, as then all the start of day code is in one place so it's easier to find.